0a1,17
> ==46203== Invalid read of size 8
> ==46203==    at 0x109FD0: infoCadena(unsigned int, _rep_cadena*) (cadena.cpp:99)
> ==46203==    by 0x10A21C: infoEnColCadenas(unsigned int, unsigned int, _rep_colCadenas*) (colCadenas.cpp:53)
> ==46203==    by 0x109C8B: main (principal.cpp:207)
> ==46203==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
> ==46203== 
> ==46203== 
> ==46203== Process terminating with default action of signal 11 (SIGSEGV)
> ==46203==  Access not within mapped region at address 0x0
> ==46203==    at 0x109FD0: infoCadena(unsigned int, _rep_cadena*) (cadena.cpp:99)
> ==46203==    by 0x10A21C: infoEnColCadenas(unsigned int, unsigned int, _rep_colCadenas*) (colCadenas.cpp:53)
> ==46203==    by 0x109C8B: main (principal.cpp:207)
> ==46203==  If you believe this happened as a result of a stack
> ==46203==  overflow in your program's main thread (unlikely but
> ==46203==  possible), you can try to increase the size of the
> ==46203==  main thread stack using the --main-stacksize= flag.
> ==46203==  The main thread stack size used in this run was 8388608.
12,33c29,39
< 12>La cantidad en la cadena 0 es 4.
< 13>La cantidad en la cadena 5 es 4.
< 14>La cantidad en la cadena 9 es 2.
< 15>La primera aparición de 3 en la cadena 0 es (3,13.10)
< 16>Se removió la primera aparición de 3 de la cadena 0.
< 17>3 está en la cadena 0.
< 18>La primera aparición de 6 en la cadena 5 es (6,6.00)
< 19>Se removió la primera aparición de 6 de la cadena 5.
< 20>6 no está en la cadena 5.
< 21>Se removió la primera aparición de 3 de la cadena 5.
< 22>Se removió la primera aparición de 4 de la cadena 9.
< 23>Se removió la primera aparición de 3 de la cadena 9.
< 24>La cantidad en la cadena 0 es 3.
< 25>La cantidad en la cadena 5 es 2.
< 26>La cantidad en la cadena 9 es 0.
< 27>Se insertó (3,3.00) en la cadena 0.
< 28>La primera aparición de 3 en la cadena 0 es (3,3.00)
< 29>La cantidad en la cadena 0 es 4.
< 30>Se insertó (6,16.10) en la cadena 5.
< 31>6 está en la cadena 5.
< 32>La primera aparición de 6 en la cadena 5 es (6,16.10)
< 33>Fin.
---
> 12>La cantidad en la cadena 0 es 0.
> 13>La cantidad en la cadena 5 es 0.
> 14>La cantidad en la cadena 9 es 0.
> 15>La primera aparición de 3 en la cadena 0 es ==46203== 400 (240 direct, 160 indirect) bytes in 10 blocks are definitely lost in loss record 3 of 3
> ==46203==    at 0x483BE63: operator new(unsigned long) (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
> ==46203==    by 0x109F3E: insertarAlInicio(unsigned int, double, _rep_cadena*) (cadena.cpp:71)
> ==46203==    by 0x10A1E8: insertarEnColCadenas(unsigned int, double, unsigned int, _rep_colCadenas*) (colCadenas.cpp:47)
> ==46203==    by 0x109BE9: main (principal.cpp:198)
> ==46203== 
> timeout: la orden monitorizada ha volcado un 'core'
> Segmentation fault
